### ISSUE TITLE
Can not config the state of systemconfigurator service as auto

### ISSUE THREAD

**Issue by v-liujxu** [ms-iot/iot-core-azure-dm-client#223]
I have built the Device Management Binaries successfully, copied _SystemConfigurator.exe_ into my Raspberry Pi3 running with Windows IoT Core 16299, and then installed the service successfully. I wanted to set the service as auto-start when device reboots using the following command:

> sc.exe config systemconfigurator start=auto

The response of this command is :

> [SC] ChangeServiceConfig SUCCESS

But when reboot the device, I checked the status of the _SystemConfigurator service through the command like:

> sc.exe query systemconfigurator

The service does not seem to really auto start.

> SERVICE_NAME: systemconfigurator
        TYPE               : 10  WIN32_OWN_PROCESS
        STATE              : 1  STOPPED
        WIN32_EXIT_CODE    : 1067  (0x42b)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0

**Comment by bderuijsscher** [comment]
As an addition, the service _does_ start when manually executing the following command(s) using a Powershell session:

```
systemconfigurator.exe -install
net start systemconfigurator
```

However, installing this service to auto start upon boot fails.

I have diagnosed this issue using IoT Core 16299 on a Raspberry PI 3.

**Comment by gmileka** [comment]
Thank you for reporting this issue... I am able to repro this problem. Initial investigation shows that the process actually starts but fails early on and crashes. I am taking a closer look...

**Comment by gmileka** [comment]
A fix has been checked-in (55fd2a34e211efdc5934815f7f1fa26a1929d9e9).
